tests/test_xopen.py

```diff
@@ -117,6 +117,14 @@ def test_pipedgzipreader_textiowrapper():
             wrapped = io.TextIOWrapper(f)
             assert wrapped.read() == CONTENT
 
+def test_bytetested_gzip():
```

Comment:
How about `test_detect_gzip_file_format_from_content` as function name? It’s fine if test function names are long and this would express clearer what the test does. Same for the other two tests.

Reply:
Changed in latest commit